org.eclipse.dawnsci.hdf5.model.test/src/org/eclipse/dawnsci/hdf5/model/internal/HierarchicalDataFileModelTest.java

```diff
@@ -306,35 +319,38 @@ public void testReadDatasetFirst() {
 				model.getPath("/entry1/instrument/analyser/cps@units"));
 	}
 
-	private Object createNexusFile(final int dtype, final long[] shape, final Object buffer) throws Exception {
+	private Object createNexusFile(final int dtype, final int[] shape, final Object buffer) throws Exception {
 		
-		String PATH = "kichwa1";
+		String PATH = Node.SEPARATOR + "kichwa1";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)